src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationAbandonTest.kt

```diff
@@ -0,0 +1,156 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.cas2
+
+import com.ninjasquad.springmockk.SpykBean
```

Comment:
I've split these tests out from `Cas2ApplicationTest` as that file is getting so big its unreadable imo...I think there's a bit of precedence with this in the other CAS tea,s with the `Person` integration tests.